server/index.js

```diff
@@ -1,58 +1,118 @@
+    PG.getMessages(req.params.id, (err, data) => {
+      if (err) {
+        res.status(500).send(err);
+      } else {
+        res.status(200);
+        res.send(data);
+      }
+    });
   });
-});
-
-app.post('/messages', (req, res) => {
-  db.createMessage(req.body, (err, result) => {
-    if (err) {
-      res.status(500).send(err);
-      return;
-    }
-    res.status(201).json(result);
+  
+  app.post('/messages', function posting (req, res) {
+    // console.log(req.body);
+    PG.createMessage(req.body, (err, result) => {
+      if (err) {
+        newrelic.noticeError(err);
+        res.status(500).send(err);
+        return;
+      }
+      res.status(201).json(result);
+    });
   });
-});
-
-app.put('/messages', (req, res) => {
-  db.updateMessage(req.body, (err, result) => {
-    if (err) {
-      res.status(500).send(err);
-      return;
-    }
-    res.status(202).json(result);
+  
+  app.put('/messages', (req, res) => {
+    PG.updateMessage(req.body, (err, result) => {
+      if (err) {
+        res.status(500).send(err);
+        return;
+      }
+      res.status(202).json(result);
+    });
   });
-});
-
-app.delete('/messages', (req, res) => {
-  db.deleteMessage(req.body, (err, result) => {
-    if (err) {
-      res.status(500).send(err);
-      return;
-    }
-    res.status(200).json(result);
+  
+  app.delete('/messages', (req, res) => {
+    PG.deleteMessage(req.body, (err, result) => {
+      if (err) {
+        res.status(500).send(err);
+        return;
+      }
+      res.status(200).json(result);
+    });
   });
-});
-
-app.listen(Port, () => {
-  console.log('Listening on port', Port);
-});
+  
+  app.get('/listings/*', (req, res) => {
+    res.sendFile(path.join(__dirname, '../public/index.html'));
+  });
+  
+  // app.get('/messages/:id', (req, res) => {
+  //   console.log(req.params.id)
+  //   req.params.id = req.params.id || 1;
+  //   db.getMessages(req.params.id, (err, data) => {
+  //     if (err) {
+  //       res.status(500).send(err);
+  //     } else {
+  //       res.status(200);
+  //       res.send(data);
+  //     }
+  //   });
+  // });
+  
+  // app.post('/messages', (req, res) => {
+  //   db.createMessage(req.body, (err, result) => {
+  //     if (err) {
+  //       res.status(500).send(err);
+  //       return;
+  //     }
+  //     res.status(201).json(result);
+  //   });
+  // });
+  
+  // app.put('/messages', (req, res) => {
+  //   db.updateMessage(req.body, (err, result) => {
+  //     if (err) {
+  //       res.status(500).send(err);
+  //       return;
+  //     }
+  //     res.status(202).json(result);
+  //   });
+  // });
+  
+  // app.delete('/messages', (req, res) => {
+  //   db.deleteMessage(req.body, (err, result) => {
+  //     if (err) {
+  //       res.status(500).send(err);
+  //       return;
+  //     }
+  //     res.status(200).json(result);
+  //   });
+  // });
+  
+  app.listen(Port, () => {
+    console.log('Listening on port', Port);
+  });
+// }
```

Comment:
Everything looks good. The only recommendation that I would make is to refactor your DB queries. Especially before we deploy to a live EC2. Good job.